src/containerapp/azext_containerapp/_help.py

```diff
@@ -126,6 +126,9 @@
     - name: Create a container app from an image in a registry with ingress enabled and a specified environment
       text: |
           az containerapp up -n MyContainerapp --image myregistry.azurecr.io/myImage:myTag --ingress external --target-port 80 --environment MyEnv
+    - name: Create a container app from an image in a registry on Connected cluster
+      text: |
+          az containerapp up -n MyContainerapp --image myregistry.azurecr.io/myImage:myTag --connected-cluster-id MyConnectedClusterResourceId
```

Comment:
Should we provide more command line here? what about the custom location related command?

Reply:
Provide this connected-cluster-id command first.